configure-ironic.sh

```diff
@@ -45,3 +49,16 @@ EOF
 
 mkdir -p /shared/html
 mkdir -p /shared/ironic_prometheus_exporter
+
+if [ "$USE_HTTP_BASIC" = "true" ]; then
+
+	crudini --set /etc/ironic/ironic.conf DEFAULT auth_strategy http_basic
+	crudini --set /etc/ironic/ironic.conf DEFAULT http_basic_auth_user_file /shared/htpasswd-ironic
+	crudini --set /etc/ironic/ironic.conf json_rpc auth_strategy http_basic
+	crudini --set /etc/ironic/ironic.conf json_rpc http_basic_auth_user_file /shared/htpasswd-ironic
+	crudini --set /etc/ironic/ironic.conf json_rpc http_basic_username $IRONIC_HTTP_BASIC_USERNAME
```

Comment:
By the way, this is incorrect: should be just `username` and `password`

Reply:
Oh damn, we're using non-standard fields for JSON RPC. This is bad...

Reply:
well according to our docs it's correct https://docs.openstack.org/ironic/latest/configuration/config.html#json-rpc .-.

Reply:
Yeah, I'm fixing this now. No clue why we let it slip... Could you add username/password to make it future-proof?

Reply:
Totally!